odoo_tools/utils/misc.py

```diff
@@ -46,3 +47,24 @@ def parse_ini_cfg(ini_content, header):
 def get_ini_cfg_key(cfg_content, header, key):
     cfg = parse_ini_cfg(cfg_content, header)
     return cfg.get(header, key)
+
+
+def get_docker_image_commit_hashes():
```

Comment:
Extracted from:
- https://github.com/camptocamp/odoo-project-tools/pull/38

Reply:
I just merged #38 you may want to rebase.

Reply:
Thanks! It's done

Notice I've made a few changes on your code